entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/Agent.java

```diff
@@ -4,7 +4,7 @@
 import java.util.List;
 import java.util.Map;
 
-public interface Agent extends Entity {
+public interface Agent extends EntityRoot {
```

Comment:
Is it really needed to change the EnityInterface to EntityRoot?

Reply:
Please revert back the renaming.

Reply:
Done.